src/core/gui/toolbarMenubar/model/ColorPalette.h

```diff
@@ -133,17 +133,17 @@ struct Palette {
     fs::path filepath;
 
     /**
-     * @brief Vector containing all colors of the palette
```

Comment:
Note: This change fixed a clang-tidy warning in the file, but is otherwise not relevant for the change.